lib/types/beacon_chain/execution_payload_deneb.ex

```diff
@@ -0,0 +1,85 @@
+defmodule Types.ExecutionPayloadDeneb do
+  @moduledoc """
+  Struct definition for `ExecutionPayload`.
+  Related definitions in `native/ssz_nif/src/types/`.
+  """
+  @behaviour LambdaEthereumConsensus.Container
+
+  fields = [
+    :parent_hash,
+    :fee_recipient,
+    :state_root,
+    :receipts_root,
+    :logs_bloom,
+    :prev_randao,
+    :block_number,
+    :gas_limit,
+    :gas_used,
+    :timestamp,
+    :extra_data,
+    :base_fee_per_gas,
+    :block_hash,
+    :transactions,
+    :withdrawals,
+    :blob_gas_used,
+    :excess_blob_gas
```

Comment:
We need to add the new fields in `docs/specs/beacon-chain.md`.

Reply:
Should we add those now or when we update the whole docs to the latest fork? I opened #775 for that

Reply:
Ah, I didn't know that was pending, we can leave it for then.